app/workers/functions.php

```diff
@@ -153,10 +161,8 @@
                 ->setAttribute('statusCode', $th->getCode())
                 ->setAttribute('stderr', $th->getMessage());
 
-            Console::error($th->getTraceAsString());
-            Console::error($th->getFile());
-            Console::error($th->getLine());
-            Console::error($th->getMessage());
```

Comment:
why did we remove this ? I think this should still stay

Reply:
At the end of job we re-throw error if one was cought. That goes to worker.php into ::error(). There see send error to sentry, but also log error.

This was removed because same error was printed to CLI twice

Reply:
okay 👍